CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
     -   `forall` instructions;
     -   `(condition).check()`-like expressions;
     -   `rename` instructions.
+-   Improved definition provider. `Go To Definition` on a method name gives the list of the locations where the method is declared and where it is implemented.
```

Comment:
So it can be declared in several places? Hum… D’you mean, like, not technically the same method, but homonyms throughout the workspace, for example? Or is this a YML feature consisting in declaring the same symbol multiple times?